modules/apps/object/object-test/src/testFunctional/paths/ObjectRootModel.path

```diff
@@ -0,0 +1,21 @@
+<html>
+<head>
+<title>ObjectRootModel</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">ObjectRootModel</td></tr>
+</thead>
+
+<tbody>
+	<tr>
+		<td>DROPDOWN_MENU_WITH_INDEX</td>
```

Comment:
Yes, I found generic paths for a dropdown menu, but to make [this macro](https://github.com/liferay-bpm-qa/liferay-portal/pull/1199/files#diff-f5e4d02c87df108d14fbfe54ab65ddc41ec9a5073871ff3879a35d78ee5f4c73R4), I needed a path that locates a dropdown menu using an index to make an increment [here](https://github.com/liferay-bpm-qa/liferay-portal/pull/1199/files#diff-f5e4d02c87df108d14fbfe54ab65ddc41ec9a5073871ff3879a35d78ee5f4c73R22)